### ISSUE TITLE
Fanboy Ultimate List not loading in uBO

### ISSUE THREAD

**Issue by Veeno** [ryanbr/fanboy-adblock#1526]
Trying to install the Fanboy Ultimate List in uBO results in this:

![image](https://user-images.githubusercontent.com/22528767/200150337-7854715d-0e73-48f3-8f45-96023a0f571a.png)

Clicking on "Subscribe" anyway results in this:

![image](https://user-images.githubusercontent.com/22528767/200150372-e759f4bb-d971-4d74-8672-6626f9658674.png)

The error shown in the uBO logger is:
`https://www.fanboy.co.nz/r/fanboy_newsletter_specific_uBO.txt?_=7: 404 Not Found`

**Comment by Cade66** [comment]
Same here. Latest successful update was ;

```
[Adblock Plus 2.0]
! Checksum: Lm4AlNCbvBZjaj+qpiI5rQ
! Title: Fanboy+Easylist-Merged Ultimate List
! Version: 202211050833
! Last modified: 2022-11-05 08:33 UTC
```
At this time uBO states as well "_A network error prevented the resource from being updated_"

**Comment by Veeno** [comment]
The workaround for now is to individually add the constituents of the Fanboy Ultimate List:

![image](https://user-images.githubusercontent.com/22528767/200169822-2b0996d5-9955-44d4-b972-e0b226ea7245.png)

**Comment by ryanbr** [comment]
Will look tomorrow.

**Comment by Veeno** [comment]
I tried it just now and it seems to be working again. Indeed, trying to open [the URL which previously resulted in a 404 error](https://www.fanboy.co.nz/r/fanboy_newsletter_specific_uBO.txt?_=7) (I did try manually opening it in the browser to check) now returns content again.

I'm not sure if it resolved itself on its own or if you fixed it @ryanbr , however, I think it's worth for this issue to stay open for now because this seems to be an intermittent problem. I remember noticing a network error for the Ultimate list previously, however in all previous instances the error was resolved by simply manually attempting to perform the update again, so I can't be sure how similar and related those issues were to this one.

I do not check the status of the filter lists very frequently, so if this is a problem that keeps intermittently appearing and resolving itself, it's possible that in all previous instances I had simply checked and performed a manual update only after the issue had already resolved itself, and this is the first time I tried to do it while the issue was still ongoing.

**Comment by Cade66** [comment]
**1- Not working here at this time.**

@Veeno wrote, 

> The workaround for now is to individually add the constituents of the Fanboy Ultimate List

_EasyList_, _EasyPrivacy_, _Fanboy's Annoyance List_ are included in uBO 1.44.4 native Filter lists, but **not** _Fanboy's Enhanced Tracking List_
Fanboy's Enhanced Tracking List can be installed from [Fanboy's Adblock Homepage](https://www.fanboy.co.nz/filters.html)

Works this way.

Confirming _Fanboy Ultimate List_ inclusions:

> Fanboy Ultimate List (This includes Easylist, Easyprivacy, Enhanced Trackers List and Annoyances List)

as stated on [Fanboy's Adblock Homepage](https://www.fanboy.co.nz/filters.html)

**2- Exact url for the Fanboy Ultimate List**

I've been using the Fanboy Ultimate List for years and the list's url was then https://secure.fanboy.co.nz/r/fanboy-ultimate.txt. Seems the url is now https://fanboy.co.nz/r/fanboy-ultimate.txt. I've always updated (until yesterday) the list with the original link. Any clue/advice to this?

**Comment by Veeno** [comment]
> 1- Not working here at this time.

Do you get an error when trying to open [the link](https://www.fanboy.co.nz/r/fanboy_newsletter_specific_uBO.txt?_=7) I included above?

If not, can you open the logger and share the error you get when trying to install/update the list?

---

> _EasyList_, _EasyPrivacy_, _Fanboy's Annoyance List_ are included in uBO 1.44.4 native Filter list

Yes, but I noticed the versions included in uBO have fewer total filters than the versions you get from https://www.fanboy.co.nz/ (possibly a slightly older version?) so I decided to use those instead of the native ones.

---

> I've been using the Fanboy Ultimate List for years and the list's url was then https://secure.fanboy.co.nz/r/fanboy-ultimate.txt. Seems the url is now https://fanboy.co.nz/r/fanboy-ultimate.txt. I've always updated (until yesterday) the list with the original link. Any clue/advice to this?

AFAIK the two domains are interchangeable, but ryanbr can shed more light on this.

**Comment by Cade66** [comment]
Yes, I do get a 404 on the [link](https://www.fanboy.co.nz/r/fanboy_newsletter_specific_uBO.txt?_=7) you mention.

I've tried again to install Fanboy Ultimate List from [Fanboy's Adblock Homepage](https://www.fanboy.co.nz/filters.html), _**Add Fanboy Ultimate Adblock list**_ : `abp:subscribe?location=https://secure.fanboy.co.nz/r/fanboy-ultimate.txt&title=Fanboy Ultimate Adblock` leads to an empty uBO's subscription page.

Now, from the url bar, I can open flawlessly both https://secure.fanboy.co.nz/r/fanboy-ultimate.txt AND
https://fanboy.co.nz/r/fanboy-ultimate.txt : the issue is all within the uBO to https://secure.fanboy.co.nz/r/fanboy-ultimate.txt connection ...

> Yes, but I noticed the versions included in uBO have fewer total filters than the versions you get from https://www.fanboy.co.nz/

Fanboy's lists are basically crafted for _AdBlock_, maybe does uBO have fewer total filters for same lists given they've been adapted for uBO? No idea really.
What I can say is that using uBO's native _EasyList_, _EasyPrivacy_, _Fanboy's Annoyance List_ in place of those offered by equivalent Fanboy's lists, together with **Fanboy only** _Enhanced Tracking List_ I get similar uBO Network Filters & Cosmetic Filters totals (+8 perhaps because last total was that of yesterday).

> AFAIK the two domains are interchangeable, but ryanbr can shed more light on this.

OK, thanks.

**Comment by Veeno** [comment]
> Yes, I do get a 404 on the [link](https://www.fanboy.co.nz/r/fanboy_newsletter_specific_uBO.txt?_=7) you mention.

Well, that'll be the error then, as well as the reason you can't install/update the filter list in uBO. That's really bizarre, because that link now loads properly for me.

**Comment by Cade66** [comment]
Would the environment have anything to do with the fact the link loads properly for you but not for me?
Here we have uBO 1.44.4 / Firefox 106.0.4 (x64) / Windows 7 (x64)

Maybe a nonsense point, just scanning imagination.

**Comment by Veeno** [comment]
> uBO 1.44.4 / Firefox 106.0.4 (x64) / Windows 7 (x64)

All same except Firefox version, which for me is 106.0.5, however, it was also already at that version when it wasn't loading, so I doubt that has anything to do with it.

**Comment by Cade66** [comment]
ryanbr mentioned,

> Will look tomorrow.

We'll know by then.
